test/integration/apiserver/admissionwebhook/admission_test.go

```diff
@@ -113,40 +113,47 @@ var (
 		gvr("", "v1", "services/proxy"): {"*": testSubresourceProxy},
 	}
 
-	// excludedResources lists resources / verb combinations that are not yet tested. this set should trend to zero.
-	excludedResources = map[schema.GroupVersionResource]sets.String{
-		// TODO: verify non-persisted review objects work with webhook admission in place (and determine whether they should be sent to admission)
-		gvr("authentication.k8s.io", "v1", "tokenreviews"):                  sets.NewString("*"),
-		gvr("authentication.k8s.io", "v1beta1", "tokenreviews"):             sets.NewString("*"),
-		gvr("authorization.k8s.io", "v1", "localsubjectaccessreviews"):      sets.NewString("*"),
-		gvr("authorization.k8s.io", "v1", "subjectaccessreviews"):           sets.NewString("*"),
-		gvr("authorization.k8s.io", "v1", "selfsubjectaccessreviews"):       sets.NewString("*"),
-		gvr("authorization.k8s.io", "v1", "selfsubjectrulesreviews"):        sets.NewString("*"),
-		gvr("authorization.k8s.io", "v1beta1", "localsubjectaccessreviews"): sets.NewString("*"),
-		gvr("authorization.k8s.io", "v1beta1", "subjectaccessreviews"):      sets.NewString("*"),
-		gvr("authorization.k8s.io", "v1beta1", "selfsubjectaccessreviews"):  sets.NewString("*"),
-		gvr("authorization.k8s.io", "v1beta1", "selfsubjectrulesreviews"):   sets.NewString("*"),
-
-		// TODO: webhook config objects are not subject to admission, verify CRUD works and webhooks do not observe them
-		gvr("admissionregistration.k8s.io", "v1beta1", "mutatingwebhookconfigurations"):   sets.NewString("*"),
-		gvr("admissionregistration.k8s.io", "v1beta1", "validatingwebhookconfigurations"): sets.NewString("*"),
+	// admissionExemptResources lists objects which are exempt from admission validation/mutation,
+	// only resources exempted from admission processing by API server should be listed here.
+	admissionExemptResources = map[schema.GroupVersionResource]bool{
+		gvr("admissionregistration.k8s.io", "v1beta1", "mutatingwebhookconfigurations"):   true,
+		gvr("admissionregistration.k8s.io", "v1beta1", "validatingwebhookconfigurations"): true,
 	}
+	// excludedResources lists resources / verb combinations that are not yet tested. this set should trend to zero.
+	excludedResources = map[schema.GroupVersionResource]sets.String{}
```

Comment:
go ahead and delete `excludedResources` and drop the code that checks it. 100% coverage 🎉

Reply:
ok, will do.. but we do not want to keep it as a place holder in case it is needed?

Reply:
I don't want a convenient way for people to skip this test :)

if we really need it in the future for some unforeseen reason, we can add it back